### ISSUE TITLE
set-output warnings in V1

### ISSUE THREAD

**Issue by tharaka-wanigasekera-ofx** [falti/dotenv-action#35]
We use falti/dotenv-action V1 in our workflows.
 
 Following is the GH action step to read env props;
```
       - name: Read environment variables from .env file
        id: dotenv
        uses: falti/dotenv-action@v1.0.0
        with:
          log-variables: true
```

However, we can see set-output warnings.
![image](https://user-images.githubusercontent.com/81727896/197935796-993c8881-dfd7-4d10-92d3-891162e50d30.png)

We would appreciate it if you could investigate this. cc @falti 

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

**Comment by sophec** [comment]
This is actually a shortcoming of the GitHub Actions Toolkit. This action uses the `setOutput` function which uses `set-output`. It looks like it's been fixed in 1.10.0. See here: actions/toolkit#1218 and actions/toolkit#1178. Updating the actions core used by this action to 1.10.0 should fix this.

**Comment by falti** [comment]
Updated core/actions - please use v1.0.1 to get rid of the warnings
